pkg/restapi/operation/didexchange/models/invitation.go

```diff
@@ -9,8 +9,9 @@ package models
 import (
 	"time"
 
+	didexchange2 "github.com/hyperledger/aries-framework-go/pkg/didcomm/protocol/didexchange"
```

Comment:
please use more descriptive names for imports :).

Reply:
done